server/src/MetricsServer.hs

```diff
@@ -0,0 +1,8 @@
+module MetricsServer where
+
+import qualified Network.Wai.Handler.Warp          as Warp
+import qualified Network.Wai.Middleware.Prometheus as PrometheusWai
+
+runMetricsServer :: Warp.Settings -> IO ()
+runMetricsServer settings = do
+  Warp.runSettings settings PrometheusWai.metricsApp
```

Comment:
I am ok with having a small module like this, but maybe more stuff could be moved here?

Reply:
Initially, I thought there would be more to the server, but it is indeed that simple. I think I'll move the function that builds the actual warp settings there.